build-info-extractor-go/src/main/java/org/jfrog/build/extractor/go/extractor/PathUtils.java

```diff
@@ -0,0 +1,78 @@
+package org.jfrog.build.extractor.go.extractor;
+
+import java.io.File;
+import java.util.regex.Pattern;
+
+/**
+ * @author BarakH
+ */
+public class PathUtils {
+
+    private static final Pattern PATTERN_SLASHES = Pattern.compile("/+");
+
+    /**
+     * @param path A file path
+     * @return Parent path of the input path as if it was a file. Empty string if the path has no parent.
+     */
+    public static String getParent(String path) {
+        if (path == null) {
+            return null;
+        }
+        File dummy = new File(path);
+        return formatPath(dummy.getParent());
+    }
+
+    /**
+     * Replaces all backslashes in the path to slashes.
+     *
+     * @param path A path to format
+     * @return The input path with all backslashes replaced with slashes. Return empty string if the input path is
+     * null.
+     */
+    public static String formatPath(String path) {
+        if (hasText(path)) {
+            path = path.replace('\\', '/');
+            return normalizeSlashes(path).toString();
+        } else {
+            return "";
+        }
+    }
+
+    /**
+     * Convert any sequence of slashses to a single slash (/)
+     *
+     * @return The normalized path
+     */
+    public static CharSequence normalizeSlashes(CharSequence path) {
+        if (path == null) {
+            return null;
+        }
+        return PATTERN_SLASHES.matcher(path).replaceAll("/");
+    }
+
+
+    /**
+     * Check whether the given CharSequence has actual text. More specifically, returns <code>true</code> if the string
+     * not <code>null</code>, its length is greater than 0, and it contains at least one non-whitespace character.
+     */
+    public static boolean hasText(String str) {
```

Comment:
I think this implementation can be simplified - 
Let's just replace  all " " with "" and return if length > 0.

Reply:
removed